frontend/src/components/LikeButtonComment/LikeButtonComment.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react'
+import axios from 'axios'
+
+import "./LikeButtonComment.scss"
+
+export default function LikeButtonComment(props) {
+    const [likeStatus, setLikeStatus] = useState(false);
+    const [likeCount, setLikeCount] = useState(props.likeCount);
+
+    useEffect(() => {
+        const getLikeStatusComment = () => {
+            axios.get(global.config.BACKEND_URL + "/comment/likeStatus/" + props.postId)
+                .then(
+                    (response) => { setLikeStatus(response.data); }
+                )
+                .catch(
+                    () => { alert("unexpected error! 🥔") }
+                )
+        }
+        getLikeStatusComment();
+    }, [likeStatus, likeCount, props.postId]) //when likeStatus change, re-render component (and call useEffect)
+
+    const likeComment = (event) => {
+        event.preventDefault();
+        likeStatus ?
```

Comment:
if your ":" for the ternary operator is at the beginning of a new line I would also make "?" the start of the line (i.e place it in front of `axios.post` instead of after `likeStatus`